src/core/components/AMInstallButton/styles.scss

```diff
@@ -59,57 +24,106 @@
     }
   }
 
-  // Preload the loading animation SVG to prevent a flicker at the start of the
-  // animation.
-  .AMInstallButton-button:hover {
-    background-image: url('./img/loading.svg');
-    background-size: 0, 0;
-  }
-
   .AMInstallButton-button--enable {
     .Icon {
       margin-top: inherit;
     }
   }
 
-  .AMInstallButton-loading {
+  .AMInstallButton-loading-button {
     background-color: $grey-20;
     border-radius: 2px;
     height: 32px;
-    text-align: center;
+  }
+
+  .AMInstallButton-loader {
+    $borderRadiusReduction: 43%;
+    $height: 4px;
+    $heightReduction: 0.9;
+    $speed: 0.53s;
+    $width: 4px;
+    $widthIncrease: 1.6;
+
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: center;
+    margin: 0 auto;
     width: 32px;
 
-    .AMInstallButton-loading-icon {
-      $width: 16px;
+    @keyframes move {
+      0% {
+        transform: translateX(0);
+      }
 
-      display: inline-block;
-      height: 4px;
-      margin-top: 14px;
-      overflow-x: hidden;
-      width: $width;
+      100% {
+        transform: translateX(12px);
+      }
+    }
+
+    [dir='rtl'] & {
+      @keyframes move-rtl {
+        0% {
+          transform: scaleX(-1) translateX(0);
+        }
+
+        100% {
+          transform: scaleX(-1) translateX(12px);
+        }
+      }
+    }
+
+    @keyframes squash {
+      0% {
+        transform: scale(1, 1);
+      }
+
+      30% {
+        border-radius: $borderRadiusReduction;
+        transform: scale($widthIncrease, $heightReduction);
+      }
+
+      70% {
+        border-radius: $borderRadiusReduction;
+        transform: scale($widthIncrease, $heightReduction);
+      }
 
-      .Icon-loading {
-        @include animated-icon(
-          $name: 'loading',
-          $steps: 60,
-          $svgWidth: 992px,
-          $width: $width
-        );
+      100% {
+        transform: scale(1, 1);
       }
+    }
+
+    .AMInstallButton-loader-container {
+      @include margin-start(8px);
+
+      animation: move $speed cubic-bezier(0.785, 0.135, 0.15, 0.86) infinite
+        alternate;
+      height: $height;
+      width: $width;
 
       [dir='rtl'] & {
-        // For some reasons, there is a small horizontal offset in RTL mode.
-        margin-left: -1px;
+        animation-name: move-rtl;
       }
     }
+
+    .AMInstallButton-loader-ball {
+      animation: squash $speed ease-in-out infinite alternate;
+      background: $blue-60;
```

Comment:
where is this defined?

Reply:
This is part of [photon](https://design.firefox.com/photon/welcome.html), which has a lib required in `src/ui/css/vars.scss`.